src/views/pages/Resources.vue

```diff
@@ -281,10 +282,18 @@ const isStaticResource = (resourceName: string): boolean => {
 };
 
 const canEditResources = (): boolean => {
-  return isAuthenticated() && hasRole(["atm", "datm", "ta", "ec", "fe", "wm"]);
+  if (permissionGroups.value.files === undefined) return false;
+  return isAuthenticated() && hasRole(permissionGroups.value.files);
 };
 
-onMounted(() => {
+onMounted(async () => {
```

Comment:
I feel this is better suited in https://github.com/adh-partnership/frontend/blob/main/src/App.vue allowing it to be used globally rather than here. Don't necessarily need to implement permissions globally now, but we can prep it now by putting this in App.vue to be called on app mounting and stored in a store that allows it to be slowly implemented elsewhere.

Reply:
I feel like that's worse than the current implementation tbh cause then were fetching on every load (regardless if they visit a page where it matters) which seems unnecessary. Ill probably move to a store and fetch if not loaded. If loaded then process for those pages.

Reply:
App.vue onMounted is only called when the initial application loads. Unless you Refresh, navigate away and come back, it isn't re-rendered.